src/install/index.js

```diff
@@ -273,15 +273,22 @@ const deploySteps = (mode, deployDoc, firstRun, ddoc) => {
       return stage('horti.stage.deploying', 'Deploying new installation')
         .then(() => performDeploy(mode, deployDoc, ddoc, firstRun))
         .then(() => {
-          stage('horti.stage.postCleanup', 'Post-deploy cleanup, installation complete');
-          return postCleanup(ddocWrapper(ddoc, mode), deployDoc);
+          return stage('horti.stage.postCleanup', 'Post-deploy cleanup, installation complete')
```

Comment:
Why is this in its own scope? I don't think it has to be. i.e. it should just be:
```js
.then(() => stage('..'))
.then(() => postCleanup(..))
```